package.json

```diff
@@ -20,7 +20,7 @@
     "flowchart"
   ],
   "author": "Jero soler",
-  "license": "MIT",
+  "license": "UNLICENSED",
```

Comment:
[request of change] The license of a fork is not possible to change it, closing PR